contrib/mac/framework/julia-config.jl

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env julia
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+const options =
+[
+    "--cflags",
+    "--ldflags",
+    "--ldlibs"
+];
```

Comment:
If we have `Julia.framework`, then the actual shared library must be named `Julia` (no lib prefix and no suffix or extension.  So unless the framework is named `libjulia.framework` (which is unconventional naming for a framework), I think there should be a mechanism for determining the name of the Julia shared library.